### ISSUE TITLE
[Feature Request] Adding custom beta version of MO2

### ISSUE THREAD

**Issue by Jtucker40** [sonic2kk/steamtinkerlaunch#922]
## System Information
- SteamTinkerLaunch version: v12.12
- Distribution: SteamOS

## Feature Description
There is a new beta version of Mod Organizer hosted on the MO discord that adds support for Starfield. Is there a way to manually have STL utilize this version even though it is not hosted on their GitHub?

**Comment by sonic2kk** [comment]
Possibly by downloading this and putting it into `~/.config/steamtinkerlaunch/downloads/mo2` and naming it `Mod.Organizer-2.4.4.exe` (and removing such an EXE if it already exists). There is no way for SteamTinkerLaunch to access files which are not publicly available, and I have no interest in adding support for files only available on a Discord server, and also the MO2 download code heavily relies on fetching version information from GitHub.

However, some points:
- Update SteamTinkerLaunch to the latest git - The installation wiki and Steam Deck wiki pages both tell you not to use v12.12 on SteamOS
    - There will not be another stable release of SteamTinkerLaunch for a very long time
- STL doesn't have Starfield in the MO2 games list yet, though since a beta is out, I can add it probably right after I post this comment.
- Doesn't MO2 have an option to enable betas in its menus?
- Do NOT expect modding Starfield to work on Linux, AT ALL. It may work, but don't expect it to, and to clear up misunderstandings other users had, **it is not up to SteamTinkerLaunch to help**
- In a similar vein, the installation may just outright fail under Wine, as any new release of any third party tool might

I have plans to introduce the ability to select the MO2 (and Vortex) installer EXEs in the future, but it is not implemented yet.

Given that this is only available on a Discord server, go in expecting to have to tinker to try and get this to work. I don't know what it might need to work if it doesn't work, and this isn't a support forum for modding on Linux.

<hr>

In short, you should be able to trick STL into using that executable if you name it, but even if it sees this, do NOT expect ANY of this to work under Wine. If you're not experienced with tinkering with tools under Wine, or if it's too much of a hassle to do on a handheld (I'm of the latter opinion, which is part of why I am opposed to modding on SteamOS), my recommendation is to avoid this altogether.

Good luck though!

**Comment by sonic2kk** [comment]
Starfield has been added to `mo2games.txt` (https://github.com/sonic2kk/steamtinkerlaunch/commit/85975d7bd87838e95eeb76395206c49d8a62ac49), but as I don't own it and have no plans to buy the game, it is not tested. Plus, even if it is detected, this MO2 build may not see it, and I doubt that both MO2 and the game will function under Wine.
